### DISCUSSION TITLE
Please invite me

### DISCUSSION THREAD

**Discussion by S-codes14** [EddieHubCommunity/support#1729] [First-Timers] [answered]
Can you add me to your organisation, I actually found you on YouTube and I have slowly become interested.

**Comment by schmelto** [comment, marked as answer]
You can use following link to create an GitHub Issue which will automatically invite you to the Github Organization :)

https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization
